packages/app/src/components/VisualizationTypeSelector/VisualizationTypeSelector.js

```diff
@@ -129,6 +131,7 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = dispatch => ({
     setUi: ui => dispatch(acSetUi(ui)),
+    clearSeriesTypes: () => dispatch(acClearSeriesType()),
```

Comment:
More general name like `onItemClick`, can be fixed later as discussed.

Reply:
Fixed